dist/templates/components/podcast.html

```diff
@@ -0,0 +1,26 @@
+									<!-- PODCAST MODULE -->
+									<table>
+                                        <tr>
+                                            <td style="vertical-align: top; width: 40%;">
+                                                <div class="content" style="padding: 25px 0 0 20px;">
+                                                    <p><img src="https://media.sailthru.com/64d/1k7/7/6/64a70725dcba6.jpg" width="230" border="0" alt="Lorem ipsum." />
+													<div style="float:left;">June 6, 2023</div> <div style="float:right;">46:05 mins</div></p><br /><br />
+												<table width="100%" border="0" cellspacing="0" cellpadding="0" style="border-radius: 0 0 5px 5px; padding-top:15px;">
+													<tr>
+														<td align="center" bgcolor="#ef404d">
+															<a role="button" class="button-text" href="{affiliate_website}" style="font-family: Arial, sans-serif; color: #ffffff; padding: 12px 18px; border: 1px solid #ef404d; display: inline-block; font-size: 18px; font-weight: bold; text-align: center; text-decoration: none; text-transform: uppercase">&#9658; Listen</a>
+														</td>
+													</tr>
+												</table>
+												</div>
+                                            </td>
+                                            <td>
+                                                <div class="content" style="padding: 0 20px 0 20px;">
+                                                    <h2>Lorem ipsum dolor sit amet</h2>
```

Comment:
is this is a new class you created, please add this at the top of this file with a <style> tag. I will reword the head style when working on mobile.